app/labor/markdown_parser.rb

```diff
@@ -119,10 +119,12 @@ def remove_nested_linebreak_in_list(html)
   end
 
   def escape_liquid_tags_in_codeblock(content)
-    # Escape BOTH codeblock and inline code
-    content.gsub(/`{3}.*?`{3}|`{1}.+?`{1}/m) do |codeblock|
-      if codeblock.include?("```")
+    # Escape codeblocks, code spans, and inline code
+    content.gsub(/`{3}.*?`{3}|`{2}.+?`{2}|`{1}.+?`{1}/m) do |codeblock|
+      if codeblock[0..2] == "```"
```

Comment:
Note that this will check for the first three characters instead of using `.include?`